internal/data/wallets.go

```diff
@@ -169,3 +320,51 @@ func (w *WalletModel) GetOrCreate(ctx context.Context, name, homepage, deepLink,
 
 	return &wallet, nil
 }
+
+func (w *WalletModel) GetCountries(ctx context.Context, walletID string) ([]Country, error) {
+	const query = `
+		SELECT
+			c.code,
+			c.name
```

Comment:
We don't need to get the country's created_at, updated_at and deleted_at dates, do we?

Reply:
At least deleted_at would be good to be returned since we're not filtering deleted records out from these lists ATM.

I agree some fields are not needed for the FE we have right now, but remember this application will be used for some partners as API-only, and keeping response objects consistent is usually easier to handle on the client side. Also, I don't think there's a relevant cost related to adding all fields.

From my experience, when we cherry-pick some fields manually like this, we can easily forget to update them if we add new fields in the future. So returning everything will prevent us from falling into that. Meaning it's easier to maintain.